build.make

```diff
@@ -185,24 +185,24 @@ $(CMDS:%=push-multiarch-%): push-multiarch-%: check-pull-base-ref build-%
 		done; \
 		docker manifest push -p $(IMAGE_NAME):$$tag; \
 	}; \
-	if [ $(PULL_BASE_REF) = "master" ]; then \
+	if [ $${PULL_BASE_REF} = "master" ]; then \
```

Comment:
You don't need `$$`, a single `$` is enough. Or is this intentional because you want to see `${PULL_BASE_REF}` in the make output when it dumps the command?

Reply:
I see, I thought I'd need to escape `$` with `$$` inside a Makefile target but I understand that variable expansion will happen before, I've updated the PR to use a single `$`